service-loadbalancer/service_loadbalancer.go

```diff
@@ -370,6 +374,12 @@ func (lbc *loadBalancerController) getServices() (httpSvc []service, tcpSvc []se
 				}
 			}
 
+			// By default sticky session is disabled
+			newSvc.SessionAffinity = false
+			if s.Spec.SessionAffinity != "None" {
```

Comment:
> can we get cookie based stickyness as well?

Yes is possible. I will add the key `serviceloadbalancer/lb.cookie-sticky-session`
If the service uses `SessionAffinity` and the annotation `serviceloadbalancer/lb.cookie-sticky-session` it will use cookies.